app/src/main/java/android/app/lotus/view/account/SupportView.kt

```diff
@@ -22,10 +22,13 @@ import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import androidx.lifecycle.ViewModel
 
+class SupportViewModel : ViewModel(){
```

Comment:
- ViewModel needs to implement @Inject constructor for Hilt/Dagger conformance
- ViewModel should be moved to Observables/ folder